src/cli.ts

```diff
@@ -139,6 +141,25 @@ export function runTranspile(files: string[], options: CliOptions) {
   }
 }
 
+function createCairoProject(filePath: string): void {
+  // create cairo_project.toml
+  const projectRoot = path.dirname(__dirname);
+  const cairoProjectPath = path.join(path.dirname(filePath), '/cairo_project.toml');
+  outputFileSync(
+    cairoProjectPath,
+    endent`
+      [crate_roots]
+      root = "."
+      warplib = "${projectRoot}/warplib"
+      maths = "${projectRoot}/warplib/maths"
+    `,
```

Comment:
It could be module level constant, since project.toml contents are the same for all outputs throughout the run

Reply:
Actually... should they? Is projectRoot absolute or relative path?

Reply:
Yeah, during the run it won't change, so it's reasonable to calculate it once on the module level

Reply:
Done

Reply:
I meant that you can make this module level constant: 
```
const CAIRO_PROJECT_TOML_CONTENT = endent`
  [crate_roots]
  root = "."
  warplib = "${warplibRoot}"
`
```